Juanma/bomb model

mirola777 opened the pull request.

mirola777 committed:
Bomb model

mirola777 committed:
Bomb class filled

mirola777 committed:
Corrección bomb. Ahora con los tipos de datos.

mirola777 committed:
Bomb controller, bomb interfaces

mirola777 committed:
Merge branch 'main' of https://github.com/Drew138/nuke-store into juanma/bomb-model

mirola777 committed:
web routes to bomb

mirola777 committed:
bomb changes

mirola777 committed:
Merge branch 'main' of https://github.com/Drew138/nuke-store into juanma/bomb-model

mirola777 committed:
Laravel pint used. Project about bombs finally works. Just there are some few details to change but actually all works

mirola777 committed:
Taller 01 completado

mirola777 committed:
Correccion con Laravel Pint. Y ruso adaptado a la app.

amchp committed:
Review changes

drew138 commented on src/app/Http/Controllers/BombController.php:
```diff
@@ -0,0 +1,70 @@
+    {
+        $data = [];
+        $data['title'] = 'Bombs - '.__('app.app_name');
+        $data['subtitle'] = __('home.home_header_bomb');
```
handle title/subtitles form the views

drew138 commented on src/app/Http/Controllers/BombController.php:
```diff
@@ -0,0 +1,70 @@
+            $bomb = Bomb::findOrFail($id);
+            $data = [];
+            $data['title'] = $bomb->getName().' - '.__('app.app_name');
+            $data['subtitle'] = $bomb->getName();
```
handle title/subtitles form the views

drew138 commented on src/app/Http/Controllers/BombController.php:
```diff
@@ -0,0 +1,70 @@
+    public function create(): View
+    {
+        $data = [];
+        $data['title'] = 'Create bomb';
```
handle title/subtitles form the views

drew138 commented on src/app/Http/Controllers/HomeController.php:
```diff
@@ -3,18 +3,29 @@
 {
+    /**
+     * Returns a view to the home of the app
+     */
```
remove unnecessary comment

mirola777 committed:
Cambios graficos a toda la aplicación.

Modificación en el archivo del modelo bomb para cambiar el Validate.

Traducciones al ruso completas.

Servicio de guardados de imágenes agregado con inversión de dependencias.

mirola777 committed:
Merge branch 'main' of https://github.com/Drew138/nuke-store into HEAD
